auslib/admin/views/forms.py

```diff
@@ -111,17 +121,28 @@ def _validator(form, field):
         # empty input
         if field.data is None:
             return
-        try:
-            op, operand = get_op(field.data)
-            version = MozillaVersion(operand)
-        except ValueError:
-            raise ValidationError("ValueError. Couldn't parse version for %s. Invalid '%s' input value" % (field.name, field.name))
-        except:
-            raise ValidationError('Invalid input for %s . No Operator or Match found.' % field.name)
-        # MozillaVersion doesn't error on empty strings
-        if not hasattr(version, 'version'):
-            raise ValidationError("Couldn't parse the version for %s. No attribute 'version' was detected." % field.name)
-
+        rulesVersionList = field.data.split(",")
+        isListOfVersions = len(rulesVersionList) > 1
+        has_relational_op = False
+        for rule_version in rulesVersionList:
+            try:
+                op, operand = get_op(rule_version)
+                if isListOfVersions and op != operator.eq:
+                    has_relational_op = True
+                    break
+                version = MozillaVersion(operand)
+            except ValueError:
+                raise ValidationError("ValueError. Couldn't parse version for %s. Invalid '%s' input value"
+                                      % (field.name, field.name))
+            except:
+                raise ValidationError('Invalid input for %s . No Operator or Match found.' % field.name)
+            # MozillaVersion doesn't error on empty strings
+            if not hasattr(version, 'version'):
+                raise ValidationError("Couldn't parse the version for %s. No attribute 'version' was detected."
+                                      % field.name)
+        if has_relational_op:
+            raise ValidationError('Invalid input for %s .Relational Operators are not allowed'
```

Comment:
This would be more readable if you got rid of has_relational_op and raised this inside the if condition on line 130. Raising it so far away from the if condition makes it seem like other things may modify the value as well.